feathr-compute/src/main/java/com/linkedin/feathr/compute/SqlUtil.java

```diff
@@ -0,0 +1,41 @@
+package com.linkedin.feathr.compute;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import net.sf.jsqlparser.JSQLParserException;
+import net.sf.jsqlparser.expression.ExpressionVisitorAdapter;
+import net.sf.jsqlparser.parser.CCJSqlParserUtil;
+import net.sf.jsqlparser.schema.Column;
+
+
+/**
+ * Class for SQL utilities
+ */
+public class SqlUtil {
+  private SqlUtil() { }
+
+  /**
+   * A hack, who knows if it will work, to try to find the input feature names from a sqlExpr derived feature.
```

Comment:
Should update this comment. We shouldn't have a comment that says "who knows if it will work"